src/main/java/com/joejoe2/chat/models/MessageType.java

```diff
@@ -4,7 +4,9 @@ public enum MessageType {
   MESSAGE("MESSAGE"),
   INVITATION("INVITATION"),
   JOIN("JOIN"),
-  LEAVE("LEAVE");
+  LEAVE("LEAVE"),
```

Comment:
One more idea out there - IMO it'd be good to move LEAVE, BAN and UNBAN into a separate enum called - _ChatActions_ for instance